azure-pipelines.yml

```diff
@@ -0,0 +1,74 @@
+pool:
+  name: Azure Pipelines
+  vmImage: windows-latest
+  demands:
+  - msbuild
+  - visualstudio
+  - vstest
+
+jobs:
+- job: Build
+  variables:
+    BuildSolution: StyleCopAnalyzers.sln
+    BuildPlatform: Any CPU
+  strategy:
+    matrix:
+      Debug:
+        BuildConfiguration: Debug
+        _debugArg: '-Debug'
+      Release:
+        BuildConfiguration: Release
+        _debugArg: ''
+  steps:
+  - task: NuGetToolInstaller@0
+    displayName: 'Use NuGet 5.3.1'
+    inputs:
+      versionSpec: 5.3.1
+
+  - task: NuGetCommand@2
+    displayName: 'NuGet restore'
+    inputs:
+      restoreSolution: '$(BuildSolution)'
+      feedsToUse: 'config'
+      nugetConfigPath: 'NuGet.config'
```

Comment:
Vastly prefer `msbuildArgs: '/restore'`

Reply:
We have to restore packages from a packages.config file.

Reply:
Whoa, why is that?

Reply:
It's how we bring in the test runner and code coverage tools today.